packages/local-storage/index.tsx

```diff
@@ -0,0 +1,50 @@
+interface SafeStorage {
+   getItem(key: string): string | null;
+   setItem(key: string, value: string): void;
+   removeItem(key: string): void;
+   key(index: number): string | null;
+   clear(): void;
+}
+
+export function useSafeLocalStorage(): SafeStorage {
+   const safeLocalStorage = {
```

Comment:
I don't see any reason why we need this const to be created in here, it doesn't close over variables, it doesn't use any state. I'd put it at the top level and then just export the object.

Reply:
I suggested this pattern because I thought of JS objects as mutable, making it safer to create a new instance for each caller in case one caller mutates the object (very unlikely, but possible). Thinking about it again though, I'm not sure how es6 modules affect mutability...

TIL that default exports are "copies" and named exports are references (I think?)
https://stackoverflow.com/questions/58437520/mutable-difference-between-es6-named-and-default-exports

Reply:
If we're worried about importers messing up the things they import, doesn't that same concern apply to *every* import in the whole ecosystem?

Reply:
i guess i could have just returned without creating a variable. Do you want another pull to address this though?

Reply:
Yea I guess it does. Your suggestion seems like a nice simplification

Reply:
I think for things like this (effectively a library with no state, no hooks, no capturing, no config) a plain export makes the most sense as it's the most predictable and friendly way of exposing a library and it let's the caller know that there's no complexity under the hood.

That said, I don't think it's worth it to make a new pull to change this.